pkg/osvscanner/customgitignore/dir_test.go

```diff
@@ -0,0 +1,309 @@
+// TODO: check if there's a way to make this: `package customgitignore` instead
+package customgitignore_test
```

Comment:
```suggestion
package customgitignore_test
```

we discussed this IRL - overall it's preferable to use a dedicated `_test` package because it helps enforce you test the _public_ API; it's only when something is really complicated and deeply nested that we want to be testing against private internals